### ISSUE TITLE
[BUG] Screen reader sometimes cannot read contacts view properly

### ISSUE THREAD

**Issue by IsaMorphic** [SubverseIM/SubverseIM#461]
Sometimes, when the contacts view is active, the screen reader will begin to read certain elements as "Not Found: `SubverseIM.ViewModels.Components.ContactViewModel`". The issue needs further investigation as the cause and specific circumstances of occurence is not clear.

**Comment by IsaMorphic** [comment]
This appears to happen only when clicking an invite link while the app is in the background (Android only)
